### DISCUSSION TITLE
Multiple selection

### DISCUSSION THREAD

**Discussion by cycyx** [deckerst/aves#1123] [Q&A] [answered]
I wonder why, but I just discovered Aves this afternoon.
I've got a few questions (maybe suggestions?), so I'll make a new discussion for each of them.

I'm used to FoSS gallery multiple selection (long click a thumbnail, then long press another one and the pictures in-between are selected): is there a way to select files in a similar way in Aves or do I have to click on each file individually?

**Comment by deckerst** [comment, marked as answer]
The way you described was suggested but not implemented. In Aves, you can long press and drag, to select (or deselect) everything in between.
